spec/acceptance/clustering_spec.rb

```diff
@@ -28,9 +30,11 @@ class { 'erlang': epel_enable => true}
     it 'runs successfully' do
       pp = <<-EOS
       class { 'rabbitmq':
+        cluster                  => { 'rabbit_cluster' => { 'init_node' => $::fqdn, 'node_disc_type' => 'ram' } },
```

Comment:
Maybe should be `$fqdn` or `$facts['fqdn']`

Reply:
please use `$facts['fqdn']`